### ISSUE TITLE
chef_image current vs latest tag

### ISSUE THREAD

**Issue by pantocrator27** [someara/kitchen-dokken#98]
In attempting to get kitchen-dokken working behind my firewall, I moved the chef images from docker hub that was tagged latest.  In reading the documentation it appears that latest is used for the latest stable version vs the use of current which is "the most recent version of Chef to pass internal testing and published to the current channel".  

When firing up test-kitchen with the dokken driver, I noticed that if you used `chef_version: latest` this gets converted to using the current tag.  https://github.com/someara/kitchen-dokken/blob/master/lib/kitchen/driver/dokken.rb#L407-L410

I am wondering if this is the correct behavior?

I always thought that in testing with test-kitchen when using the latest version of chef it was the latest version from the stable channel.  

I am wondering if that behavior should mirror this or at least provide distinction between the use of latest vs current tags meaning a user could specify `chef_version: latest` or `chef_version: current`

**Comment by someara** [comment]
@tas50 how should this work?

**Comment by pantocrator27** [comment]
From the README.md:
```
Chef publishes all functioning builds to the Docker Hub, including those from the "current" channel. If you wish to use pre-release versions of Chef, set your chef_version value to "current".
```

**Comment by tas50** [comment]
@someara  The desired behavior in my mind is "stable" pulls the latest stable release and current pulls the latest current channel release.

**Comment by pantocrator27** [comment]
By stable do you mean latest according to docker tag?

**Comment by someara** [comment]
This should be fixed up in f8b062104371c23409c903fd2fa411c40a5222d8

Released in 2.1.4

-s
